src/Perpetuum/Zones/Terrains/Materials/Plants/PlantHandler.cs

```diff
@@ -21,6 +21,8 @@ public class PlantHandler : IPlantHandler
         public delegate IPlantHandler Factory(IZone zone);
 
         private const int AREA_SIZE = 32;
+        // Number of hours a full pass of plant regen should take
+        private readonly int FULL_PLANT_REGEN_PASS = 8;
```

Comment:
Following up on the previous review with the int division issue:
You could declare this as a `TimeSpan` and all operators will invoke timespan methods like:
`/` will call `TimeSpan.divide()` so you can operate on timespans directly multiplying, dividing, adding two together, etc.
https://docs.microsoft.com/en-us/dotnet/api/system.timespan.op_division?view=net-5.0